airflow/models/dagrun.py

```diff
@@ -1648,7 +1648,5 @@ def __init__(self, content, user_id=None):
         self.user_id = user_id
 
     def __repr__(self):
-        prefix = f"<{self.__class__.__name__}: {self.dag_id}.{self.dagrun_id} {self.run_id}"
-        if self.map_index != -1:
-            prefix += f" map_index={self.map_index}"
+        prefix = f"<{self.__class__.__name__}: {self.dag_run_id}.{self.content}"
```

Comment:
This information is not sufficient, we can use the `dag_run` relationship to fetch the dag_id and the run_id.

Reply:
ok，I will try it.